tests/libecalc/presentation/yaml/test_venting_emitter.py

```diff
@@ -1,18 +1,15 @@
+from copy import deepcopy
```

Comment:
Based on the usage of ltp_functions in this test, it seems that this also tests ltp. So we could move the tests to exporter/test_ltp or create a new file in exporter for test_venting_emitter_ltp. That way ltp functions can be a common fixture within exporter, instead of top-level conftest

Reply:
Some emitter tests moved to ltp, and fixture is moved back to LtpTestHelper